x/oracle/keeper/historic_price.go

```diff
@@ -296,6 +296,11 @@ func (k Keeper) AddHistoricPrice(
 	ak.updateAvgCounter(denom, exchangeRate, ctx.BlockTime())
 }
 
+func (k Keeper) GetHistoricAvgPrice(ctx sdk.Context, denom string) (sdk.Dec, error) {
```

Comment:
How about renaming to `HistoricAvgPrice(...)`?

Reply:
ok